docs/sdk/web3-react/guides/02-connectors.md

```diff
@@ -1,4 +1,159 @@
+
+The difference lies in the second argument, which is an `options` object. In this case, we are passing the `rpc` parameter, which is an object that maps the chain ID to the RPC URL to connect to. We have already created this [map](https://github.com/Uniswap/examples/blob/8c0e36ca8d2ba4718af944094191f39da62a9c5c/web3-react/src/libs/constants.ts#L11) in our [constants](https://github.com/Uniswap/examples/blob/feat/web3-react/web3-react/src/libs/constants.ts) file using our example's parameters. The other option that we are passing is the `qrcode`, which is a `boolean` that indicates whether the QR code should be displayed in the browser. In our case, we are passing `true` as the value of this option as we want to show the qr code.
+
+Having built the connector, we just need to build the user interface to enable user interaction with the connector, and supply it to our `ConnectionOptions`:
+    
+```typescript reference title="Building the WalletConnect Wallet component" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/81ec93e97b0afded621e177fe5f34fc9f98f80b0/web3-react/src/libs/components/ConnectionOptions.tsx#L49-L56
+```
+## Building a Network connector
+
+The *Network connector*, alongside the *Gnosis Safe connector*, are two of the connectors that we do not surface through our user interface, but instead we connect to them programmatically. In contrast to the previous *Connectors*, these do not come with any pre-built user interface for the user to interact with. We attempt to connect to them **eagerly** in our [`Web3ContextProvider`](https://github.com/Uniswap/examples/blob/feat/web3-react/web3-react/src/libs/components/Web3ContextProvider.tsx) component through a hook:
+    
+```typescript reference title="Hook to connect eagerly" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/81ec93e97b0afded621e177fe5f34fc9f98f80b0/web3-react/src/libs/components/Web3ContextProvider.tsx#L9-L13
+```
+
+The `useEagerlyConnect` hook is called in the `Web3ContextProvider` component, and attempts to connect to the Network Connector, and the Gnosis Safe Connector. The hook is named **eagerly**, as it is called in the component's body as an [React effect](https://reactjs.org/docs/hooks-effect.html) when the component is first rendered. In the hook implementation we attempt to call `web3-react`'s `connectEagerly` function if it exist on the connector, or call `activate` otherwise. The `connectEagerly` function attempts to connect our application to the connector, and fails silently if it does not succeed: 
```

Comment:
```suggestion
The `useEagerlyConnect` hook is called in the `Web3ContextProvider` component, and attempts to connect to the Network Connector and the Gnosis Safe Connector. The hook is named **eagerly** as it is called in the component's body as a [React effect](https://reactjs.org/docs/hooks-effect.html) when the component is first rendered. In the hook implementation we attempt to call `web3-react`'s `connectEagerly` function if it exists on the connector, otherwise we call `activate`. The `connectEagerly` function attempts to connect our application to the connector, and **fails silently** if it does not succeed: 
```